sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeFunctions.scala

```diff
@@ -54,3 +60,193 @@ case class CurrentTimestamp() extends LeafExpression {
     System.currentTimeMillis() * 1000L
   }
 }
+
+case class Hour(child: Expression) extends UnaryExpression with ImplicitCastInputTypes {
+
+  override def inputTypes: Seq[AbstractDataType] = Seq(TimestampType)
+
+  override def dataType: DataType = IntegerType
+
+  override protected def nullSafeEval(timestamp: Any): Any = {
+    DateTimeUtils.getHours(timestamp.asInstanceOf[Long])
+  }
+
+  override def genCode(ctx: CodeGenContext, ev: GeneratedExpressionCode): String = {
+    val dtu = "org.apache.spark.sql.catalyst.util.DateTimeUtils"
```

Comment:
`DateTimeUtils.getClass.getName` returns `org.apache.spark.sql.catalyst.util.DateTimeUtils$`. Is there a way to select the name automatically and don't get the `$`?